src/App/App.js

```diff
@@ -113,8 +130,16 @@ const App = () => {
         if (services.core.active) {
             onWindowFocus();
             window.addEventListener('focus', onWindowFocus);
+            services.core.transport.on('CoreEvent', onCoreEvent);
+            services.core.transport
+                .getState('ctx')
+                .then(onCtxState)
+                .catch((e) => console.error(e));
```

Comment:
Nitpicking but this can be even shorter :) 

```suggestion
                .catch(console.error);
```